dsp-repository/v1/dsp-repository.shacl.ttl

```diff
@@ -26,32 +26,670 @@
+    rdfs:subPropertyOf schema:sameAs .
+
+# http://ns.dasch.swiss/repository#hasTitle
+dsp-repo:hasTitle
+    a rdf:Property .
+
+# # http://ns.dasch.swiss/repository#hasAlternativeTitle
+# dsp-repo:hasAlternativeTitle
+#     a rdf:Property .
+
+# http://ns.dasch.swiss/repository#hasAbstract
+dsp-repo:hasAbstract
+    a rdf:Property .
+
+# http://ns.dasch.swiss/repository#hasEncodingFormat
+dsp-repo:hasEncodingFormat
+    a rdf:Property .
+
+# http://ns.dasch.swiss/repository#hasDocumentation
+dsp-repo:hasDocumentation
+    a rdf:Property .
+
+# http://ns.dasch.swiss/repository#hasLicense
+dsp-repo:hasLicense
+    a rdf:Property ;
+    rdfs:subPropertyOf schema:license .
+
+# http://ns.dasch.swiss/repository#hasConditionsOfAccess
+dsp-repo:hasConditionsOfAccess
+    a rdf:Property .
+
+# http://ns.dasch.swiss/repository#hasHowToCite
+dsp-repo:hasHowToCite
+    a rdf:Property .
+
+# http://ns.dasch.swiss/repository#hasStatus
+dsp-repo:hasStatus
+    a rdf:Property .
+
+# http://ns.dasch.swiss/repository#hasDatePublished
+dsp-repo:hasDatePublished
+    a rdf:Property .
+
+# http://ns.dasch.swiss/repository#hasLanguage
+dsp-repo:hasLanguage
+    a rdf:Property .
 
+# http://ns.dasch.swiss/repository#isPartOf
+dsp-repo:isPartOf
+    a rdf:Property ;
+    rdfs:subPropertyOf schema:isPartOf .
+
+# http://ns.dasch.swiss/repository#hasQualifiedAttribution
+dsp-repo:hasQualifiedAttribution
+    a rdf:Property ;
+    rdfs:subPropertyOf prov:qualifiedAttribution .
+
+# http://ns.dasch.swiss/repository#hasDateCreated
+dsp-repo:hasDateCreated
+    a rdf:Property .
+
+# http://ns.dasch.swiss/repository#hasDateModified
+dsp-repo:hasDateModified
+    a rdf:Property .
+
+# http://ns.dasch.swiss/repository#hasDistribution
+dsp-repo:hasDistribution
+    a rdf:Property ;
+    rdfs:subPropertyOf schema:distribution .
+
+# http://ns.dasch.swiss/repository#hasGivenName
+dsp-repo:hasGivenName
+    a rdf:Property .
+
+# http://ns.dasch.swiss/repository#hasFamilyName
+dsp-repo:hasFamilyName
+    a rdf:Property .
+
+# http://ns.dasch.swiss/repository#hasEmail
+dsp-repo:hasEmail
+    a rdf:Property .
+
+# http://ns.dasch.swiss/repository#hasAddress
+dsp-repo:hasAddress
+    a rdf:Property .
+
+# http://ns.dasch.swiss/repository#isMemberOf
+dsp-repo:isMemberOf
+    a rdf:Property ;
+    rdfs:subPropertyOf schema:memberOf .
+
+# http://ns.dasch.swiss/repository#hasJobTitle
+dsp-repo:hasJobTitle
+    a rdf:Property .
+
+# http://ns.dasch.swiss/repository#hasRole
+dsp-repo:hasRole
+    a rdf:Property ;
+    rdfs:subPropertyOf prov:hadRole .
+
+# http://ns.dasch.swiss/repository#hasNumber
+dsp-repo:hasNumber
+    a rdf:Property .
+
+# # http://ns.dasch.swiss/repository#hasBooBoo
+# dsp-repo:hasBooBoo
+#     a rdf:Property .
+
+##### CLASS DEFINITIONS #####
+### --- Class definition of dsp-repo:Project --- 
+# http://ns.dasch.swiss/repository#Project
+dsp-repo:Project 
+    a rdfs:Class ;
+    rdfs:label "dsp-repo:Project"@en ;
+    rdfs:comment "A project"@en ;
```

Comment:
I agree that we should elaborate more on the comment. In my opinion however, this can also be something to do in a later revision. (And from my point of view, thinking of the NIE-INE projects, I hope to have something finalized relatively soon.)

I'd say, if you have concrete suggestions for better comments in mind, feel free to add them and push them (we're in a work-in-progress-branch after all). But if that's something you don't want to do right away, but is rather a request for improvement, I'd suggest accepting it as it is now, and once this is merged to main, we cold open a new user story and PR that's specific to improving comments (and maybe the labels too), and work on it there.

Again, what say you, @danielasubotic ?

Reply:
What are our focus and main goal? It is NIE-INE. In the next 5 weeks (max.!) NIE-INE must have handed over all projects to us. 
What are the next steps to achieve this?
- the metadata must be ready for Balduin to use
- Balduin needs to create the metadata entry form
- NIE-INE runs a test project with it
- more tests on our side
- and many more tasks and steps...
If you ask me, the timeline is very, very tight. We hope for smooth progress, but we have to expect the unexpected.
We have now added valuable and important input from @mrivoal , @gautschr , and @BalduinLandolt  to this version. And I thank all of you for your collaboration, ideas, and suggestions.
I agree with Balduin at 200%. Any changes in the comments that are still desired should be made by yourself, @mrivoal  (pull/make it pretty/push). Otherwise User Stories.
The creation of metadata is a process and it is important that it continues and it will continue.
So, @BalduinLandolt , the deadline for the metadata is this Thursday, right?

Reply:
ideally, yes... :)  
if not, then it's just "as soon as possible"

Reply:
@BalduinLandolt "as soon as possible" sounds really stretchy to me ;-) just name a date ^^
Just think about the main goal, the timeline and your resources.

Reply:
okay... let's say we have it merged by Friday evening.  
That allows @mrivoal to finish her review tomorrow, and then we wrap up on friday.

After that, we move all improvement (which I still want to encourage!) to a new user story and branch.